src/main/resources/avro/biodatamethods.avdl

```diff
@@ -0,0 +1,147 @@
+@namespace("org.ga4gh.methods")
+
+protocol BiodataMethods {
+
+import idl "metadata.avdl";
+import idl "common.avdl";
+import idl "methods.avdl";
+import idl "biodata.avdl";
+
+/******************  /individuals/search  *********************/
+/**
+This request maps to the body of `POST /biosamples/search` as JSON.
+*/
+record SearchIndividualsRequest {
+  /**
+  The dataset to search within.
+  */
+  string datasetId;
```

Comment:
Although the dataset conceptual model is fuzzy, I believe the
intention is that objects can only belong to one data set.  If
there is a use case for objects belonging to multiple datasets,
we need to revisit this model ASAP.

David Steinberg <notifications@github.com> writes:
> In src/main/resources/avro/biodatamethods.avdl:
> 
> > +protocol BiodataMethods {
> > +
> > +import idl "metadata.avdl";
> > +import idl "common.avdl";
> > +import idl "methods.avdl";
> > +import idl "biodata.avdl";
> > +
> > +/******************  /individuals/search  *********************/
> > +/**
> > +This request maps to the body of `POST /biosamples/search` as JSON.
> > +*/
> > +record SearchIndividualsRequest {
> > +  /**
> > +  The dataset to search within.
> > +  */
> > +  string datasetId;
> 
> I had added datasetId in a previous pull request, but it was requested to leave
> datasetId out of the record itself so a sample or individual could belong to
> multiple datasets. #586
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub*
>